src/org/usfirst/frc/team694/robot/Robot.java

```diff
@@ -56,16 +46,27 @@ public void robotInit() {
         grabber = new Grabber();
         lift = new Lift();
         oi = new OI();
-        
-<<<<<<< HEAD
-       
-=======
+        currentQuad = getRobotQuadrant();
```

Comment:
Don't do this. Here we have to assume that this variable is set. Just use getRobotQuadrant() in each command.